testsuite/src/main/java/io/netty5/testsuite/transport/socket/SocketTestPermutation.java

```diff
@@ -158,4 +203,26 @@ public List<BootstrapFactory<Bootstrap>> datagramSocket() {
                 () -> new Bootstrap().group(nioWorkerGroup).channel(NioDatagramChannel.class)
         );
     }
+    public static DomainSocketAddress newDomainSocketAddress() {
+        try {
+            File file;
+            do {
+                file = PlatformDependent.createTempFile("NETTY", "UDS", null);
+                if (!file.delete()) {
+                    throw new IOException("failed to delete: " + file);
+                }
+            } while (file.getAbsolutePath().length() > 128);
```

Comment:
This could be an infinite loop if the createTmpFile consistently makes files with too long names due to the tmp dir path or temp file name format.